test/test_nn.py

```diff
@@ -8919,8 +8919,11 @@ def _test_module_empty_input(self, module, inp):
         out.backward(gO)
         self.assertEqual(out.size(), inp.size())
         for p in module.parameters():
+            # TODO: p.grad should not be None, but this is not yet supported
+            # (https://github.com/pytorch/pytorch/issues/12013)
```

Comment:
Which part of the linked issue talks about `None` grad? I couldn't find it.

Reply:
https://github.com/pytorch/pytorch/issues/12013#issuecomment-575871264 says `p.grad` is None